### DISCUSSION TITLE
SSL Certificate Invalid

### DISCUSSION THREAD

**Discussion by gagecarto** [community/community#50942] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

We've been successfully running a site off github pages for almost 1 year. The site is accessible via a url like

https://username.github.io/repoName/

As of this morning we're getting SSL errors and our certificate seems to be invalid. Any suggestions or suggested troubleshooting?

**NET::ERR_CERT_AUTHORITY_INVALID**

What is strange is that the SSL error does not occur when accessing the site through a mobile, cellular connection. It's only occurring when accessing from a computer connected to internet etc.

**Comment by yoannchaudet** [comment]
We had an outage with our CDN which was resolved promptly: https://twitter.com/githubstatus/status/1639372937332662273. More than Pages was impacted unfortunately.

**Comment by gagecarto** [comment]
Hi Yoann,

Thanks for the update on these issues. Everything is working well for me
here in US but I have clients in the UK still experiencing errors. Will
this issue take some time to resolve globally? Emma (cc'd) here is in
London and still getting errors like below.


Afraid I still can't access the github pages on wifi. (they come and go)
Would it be possible to copy me into a reply to GitHub so we can get to the
bottom of it?
I think it is down to a security setting on my wifi blocking them - and
want to understand why this is happening or if it is happening to anyone
else.
I usually get this error message:
[image: Screenshot 2023-03-27 at 16.37.38.png]

But one time I got this:
[image: Screenshot 2023-03-27 at 16.19.15.png]
Many thanks,
Emma


----------------

Emma Griffin

Co-founder, Footways

Footways.London  <http://footways.london/>

07960 587030

-----------------

On Fri, Mar 24, 2023 at 5:00 PM Yoann Chaudet ***@***.***>
wrote:

> We had an outage with our CDN which was resolved promptly:
> https://twitter.com/githubstatus/status/1639372937332662273. More than
> Pages was impacted unfortunately.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/50942#discussioncomment-5423518>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABMCK7V2UT4N3SBRBFCP5ULW5YRRFANCNFSM6AAAAAAWGZBN3Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
----
*Gage Cartographics <http://www.gagecarto.com/>*
406-570-1060

**Reply by yoannchaudet** [reply]
Please reach out to [GitHub support](https://support.github.com/contact).

**Comment by gagecarto** [comment]
Thanks Josh and it's great to be connected Yoann.
Sure enough, if I turn off 'Virus Safe' on my internet provider, the GitHub
pages work. But I'd rather find a different solution.
A couple of weeks ago, a couple of corporate or government users told us
they couldn't access our maps/pages - which now makes me think it's
something to do with their organisations' virus protection blocking the
pages.
Is there anything we can do to prevent this happening?
Many thanks,
Emma

----------------

Emma Griffin

Co-founder, Footways

Footways.London  <http://Footways.London>

07960 587030

-----------------
